isaric/parsers/ccp-drc.toml

```diff
@@ -298,6 +298,11 @@
   description = "Health worker?"
   ref = "Y/N/NK"
 
+[subject.works_lab]
+  field = "microb"
+  description = "Employee of a microbiology laboratory"
```

Comment:
@pipliggins I think for creating the works_lab supertype we'd only map the Y responses if the question specifically asks about working in a microbiology lab, as someone could respond 'N' to not working in a microbiology lab but they are still a lab worker, what do you think?

Reply:
@sadiekelly so you mean if a question asks 'are you a microbiology worker?' we'd only map the yes responses to works_lab, but both Y and N for works_microbiology_lab? Yes, I think that makes sense. We might potentially need to consider this logic for other super types as well...

Reply:
yes, that's right, I think that's the best way. Which of the other super types are you thinking of?

Reply:
I haven't thought it through... but possibly for things like oxygen therapy - @abhidg remind me, if a 'false' response is given within combinedType = any, will that override a 'True' response from earlier?